apps/web/src/modules/auction/components/CurrentAuction/GroupBidLink.tsx

```diff
@@ -0,0 +1,20 @@
+import { Flex, Text } from '@zoralabs/zord'
+import React from 'react'
+
+import { Icon } from 'src/components/Icon'
+
+interface GroupBidLinkProps {}
+
+export const GroupBidLink: React.FC<GroupBidLinkProps> = () => {
+  return (
+    <a href="https://www.grouppurple.com/" target="_blank" rel="noreferrer">
+      <Flex direction="row" justify="space-between" style={{ marginTop: '10px' }}>
+        <Icon id="sparkles" />
```

Comment:
This is meant to signify that the feature, or link, is new. But I'm happy to remove it or replace it with a different icon if preferred.